### ISSUE TITLE
Support Jammy in third_party_apps_test

### ISSUE THREAD

**Issue by martijnvans** [GoogleCloudPlatform/ops-agent#1423]
The following two platforms should pass third_party_apps_test:

ubuntu-2204-lts
ubuntu-2204-lts-arm64

currently they do not due to various small issues. Non-comprehensive list:

1. oracledb is not configured to skip these platforms (but it should).
2. cassandra fails with `GPG error: https://archive.debian.org/debian-security stretch/updates InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AA8E81B4331F7F50 NO_PUBKEY 112695A0E562B32A`
3. rabbitmq fails with "unknown version" from here: https://github.com/GoogleCloudPlatform/ops-agent/blob/96d0aac034de6c175346146a27eebf744a6ad20f/integration_test/third_party_apps_data/applications/rabbitmq/debian_ubuntu/install#L31
4. mysql 5.7 fails with E: Version '5.7*' for 'mysql-server' was not found, looks like we should probably add that to platforms_to_skip here: https://github.com/GoogleCloudPlatform/ops-agent/blob/96d0aac034de6c175346146a27eebf744a6ad20f/integration_test/third_party_apps_data/applications/mysql5.7/metadata.yaml#L35
5. dcgm fails with `ERROR: An error occurred while performing the step: "Building kernel modules". See /var/log/nvidia-installer.log for details.`, I guess we should also add these platforms to platforms_to_skip?
6. nvml similarly fails with a weird `Failed to verify gcc version. See log at /var/log/cuda-installer.log for details.` error, but probably we should just add the two jammy platforms to platforms_to_skip.
7.  aerospike fails with `err=WaitForMetricSeries(metric=agent.googleapis.com/agent/internal/ops/feature_tracking, extraFilters=[]) failed: exhausted retries`

**Comment by jefferbrecht** [comment]
`dcgm` and `nvml` should definitely be skipped on `ubuntu-2204-lts-arm64`, but I would expect it to pass on `ubuntu-2204-lts`...

**Comment by martijnvans** [comment]
Yeah, I'm kicking off a run with that exact set skipped and we can take a look together: https://github.com/GoogleCloudPlatform/ops-agent/pull/1425

**Comment by martijnvans** [comment]
Fixed by https://github.com/GoogleCloudPlatform/ops-agent/pull/1429.
